src/main/resources/project/procedures/form_scripts/validation/createConfiguration.groovy

```diff
@@ -125,6 +143,11 @@ def doValidations(args) {
 			result = buildErrorResponse("${credential[CREDENTIAL_NAME]}.$USER_NAME", "Invalid Access Key ID").
 					error("${credential[CREDENTIAL_NAME]}.$PASSWORD", "Invalid Access Key")
 		}
+
+		//TODO: Example for returning validation error against the 'proxy_credential' parameter
+		//if (proxyCredential) {
+		//	result.error("proxy_credential.userName", "Invalid Proxy User '${proxyCredential.userName}' or password '${proxyCredential.password}'")
```

Comment:
Just want to point out to be sure. Do not put the password in the response. This was just my experimental code to confirm that we were getting the password all the way through to the script.